gdc_ng_models/models/genomic_data.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+GENOMIC_DATA_TYPE_ENUM = Enum("ssm", "cnv", "case", name="genomic_data_data_type")
+
+
+class GenomicData(Base):
```

Comment:
Might want to call this `ReleasedData` since it's already expanding a little past genomic data.